tests/acceptance/30_generic_methods/service_check_running.cf

```diff
@@ -22,7 +22,12 @@ body common control
 bundle agent init
 {
   vars:
-    "service_name"  string => "cron";
+    redhat::
+      "service_name"  string => "crond";
+    aix::
+      "service_name"  string => "sshd";
```

Comment:
Given that most of the time the machine doing the test will be controlled by someone like us you can use a use a less critical service like syslogd, snmpd or inetd.